### ISSUE TITLE
Django SECRET_KEY is stored in the source code

### ISSUE THREAD

**Issue by DavidToroczkai** [lxndrvn/lemezpolc#3]
Hello @lxndrvn, thanks for the great talk yesterday about this project!
When skimming through the code I noticed that you have left the SECRET_KEY you are using in settings.py: https://github.com/lxndrvn/lemezpolc/blob/e9eda900fb5f9bc014a1e3c16d59c12ae3b40d5e/lemezpolc_project/settings.py#L23

This is dangerous as anyone knowing this key can go around Django's security protections (https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-SECRET_KEY). I would recommend storing this in the environment, the same way as you are doing it with the API keys here:
https://github.com/lxndrvn/lemezpolc/blob/57d8896f23361394147a98229534f6dd8ab0bb01/lemezpolc/jobs/scrape_discogs_data.py#L8

Also make sure to change the secret key in your local project, as this key is now insecure!

**Comment by lxndrvn** [comment]
Thanks for the feedback, it's really useful! I'll move the secret key to the environment.

**Comment by markkovari** [comment]
Yeah, sorry that was mine too, but I was on the tram and the description of the issue was so short. :(   Sorry for the inconvenience. Later I'll try to write a bit more obvious issues and anything else. I just want to keep your keys safe 🐱
